### PR TITLE
[Property][Unittest] Mixed Precision Unittest & Adding verification code @open sesame 12/02 09:00

### PR DESCRIPTION
**Pull request by DonghakPark**
## In this Pull Request

I would like to add missing verification code related to mixed precision operation and also include associated unittest 
- test each layers compile, initialize, train 
- add some property's validation check code

**Self evaluation:**
1. Build test:   [X]Passed [ ]Failed [ ]Skipped
2. Run test:     [X]Passed [ ]Failed [ ]Skipped
    
Signed-off-by: Donghak PARK <donghak.park@samsung.com>

### CODE REVIEW

**Review by skykongkong8 — APPROVED** [review]
LGTM!

File: nntrainer/layers/common_properties.h [hunk 1]
```diff
@@ -1462,6 +1462,15 @@ class LossScaleForMixed : public Property<float> {
 public:
   static constexpr const char *key = "loss_scale"; /**< unique key to access */
   using prop_tag = float_prop_tag;                 /**< property type */
+
+  /**
+   * @brief check if given value is valid
+   *
+   * @param v value to check
```
Comment (EunjuYang) [inline nntrainer/layers/common_properties.h:1469]:
```suggestion
   * @param value value to check
```

Comment (DonghakPark) [reply nntrainer/layers/common_properties.h:1469]: Thank you !! i will fix

**Review by EunjuYang — APPROVED** [review]
LGTM!

**Pull request merged**